Docs/Configuration.tex

```diff
@@ -3092,16 +3092,30 @@ \subsection{Boot Properties}\label{miscbootprops}
       \item \texttt{<ENTRY\_PATH>.icns} --- icon near the entry file with appended \texttt{.icns} extension.
     \end{itemize} \medskip
 
-    For all other entries OpenCore will try to load a volume icon and fallback to the default icon:
+    For all other entries OpenCore will try to load a volume icon by searching
+    as follows, and will fallback to the default icon otherwise:
     \begin{itemize}
     \tightlist
-      \item \texttt{.VolumeIcon.icns} file at \texttt{Preboot} volume directory for APFS (if present).
-      \item \texttt{.VolumeIcon.icns} file at \texttt{Preboot} root for APFS (otherwise).
+      \item \texttt{.VolumeIcon.icns} file at \texttt{Preboot} per-volume directory
+      (\texttt{/System/Volumes/Preboot/-GUID-/} within macOS) for APFS (if present).
+      \item \texttt{.VolumeIcon.icns} file at \texttt{Preboot} root
+      (\texttt{/System/Volumes/Preboot/} within macOS) for APFS (otherwise).
       \item \texttt{.VolumeIcon.icns} file at volume root for other filesystems.
     \end{itemize} \medskip
+    
+    \emph{Note 1}: Apple's boot picker partially supports placing a volume icon file
+    at \texttt{Data} root (\texttt{/System/Volumes/Data/} within macOS). This approach
```

Comment:
Here it also makes think that `Data` volume, just like `Preboot`, is a single volume within the container. But it is rather: Preboot, Recovery, System1, Data1, System2, Data2, System3, Data3. Maybe something like this:
```
    \emph{Note 1}: Apple's boot picker partially supports placing a volume icon file
     at operating system's \texttt{Data} volume root (\texttt{/System/Volumes/Data/}
     when mounted at default location within macOS). This approach
```

Reply:
Added word "the" ("the operating system's") and applied 👍